app/recordtransfer/management/commands/schedule_jobs.py

```diff
@@ -0,0 +1,39 @@
+import logging
+
+import django_rq
+from django.conf import settings
+from django_rq.management.commands import rqscheduler
+
+from recordtransfer.jobs import cleanup_expired_sessions
+
+scheduler = django_rq.get_scheduler()
+LOGGER = logging.getLogger("rq.scheduler")
```

Comment:
Don't need a special logger here, the `recordtransfer` logger setup will capture this log.

```suggestion
LOGGER = logging.getLogger(__name__)
```

Reply:
Just to make sure I understand this fully - is your solution to let the `recordtransfer` logger capture the logs we made, and let `rq_scheduler` and `rq_scheduler.scheduler` intercept the logs made from the `rq_scheduler` package?

If so:
- could you explain why we would want to intercept the logs made from the package in the first place, if they already show up without intercepting them?
- why are not following the same pattern for different loggers within the app such as in [`emails.py`](https://github.com/NationalCentreTruthReconciliation/Secure-Record-Transfer/blob/3f4f4972a752adabee19074f37da76b84abce8ac/app/recordtransfer/emails.py#L21), where the logger used is `rq.worker` and not `__name__` (which would resolve to the `recordtransfer` logger)

Reply:
> could you explain why we would want to intercept the logs made from the package in the first place, if they already show up without intercepting them?
Ah I think this is because we want a consistent format for all log messages, such as the log level and the timestamp.

Reply:
Addressed by: https://github.com/NationalCentreTruthReconciliation/Secure-Record-Transfer/pull/403/commits/2beb16f2f1e62f77027debbadd860407e2f3b172

Reply:
> Just to make sure I understand this fully - is your solution to let the `recordtransfer` logger capture the logs we made, and let `rq_scheduler` and `rq_scheduler.scheduler` intercept the logs made from the `rq_scheduler` package?

Correct, yes.

>  * could you explain why we would want to intercept the logs made from the package in the first place, if they already show up without intercepting them?

This is to provide a uniform log output in the container and provide control over the logs. If we want to write all the logs to a file in the future, we can simply add another handler in our config.

>  * why are we not following the same pattern for different loggers within the app such as in [`emails.py`](https://github.com/NationalCentreTruthReconciliation/Secure-Record-Transfer/blob/3f4f4972a752adabee19074f37da76b84abce8ac/app/recordtransfer/emails.py#L21), where the logger used is `rq.worker` and not `__name__` (which would resolve to the `recordtransfer` logger)

This is because of a misunderstanding I made about how logging works when I first wrote this app. All loggers should ideally look like this: `LOGGER = logging.getLogger(__name__)`.

I understand now that each process gets its own logging setup defined by the `dictConfig`. My mistake came from thinking that I would have to log to `rq.worker` to get the logs to show up in the `rq` container. I know now that there's a separate `recordtransfer` logger that runs in each `app`, `rq`, and now `rq-scheduler`. So each of those apps can write logs to `recordtransfer` and they'll get written to the logs of their respective containers.

Similarly with the emails, I thought those would have to get logged to `rq.worker`, since the emails are asynchronous jobs. No need to fix all the instances of bad logging setup right now, but we may as well fix the one I identified right now because that logger gets called in your new `cleanup_expired_sessions` function.

Reply:
Things are much clearer now - thank you very much for the detailed explanation!

Reply:
For sure, thanks for asking!